modules/scala-http4s/src/main/scala/dev/guardrail/generators/scala/http4s/Http4sServerGenerator.scala

```diff
@@ -739,13 +739,11 @@ class Http4sServerGenerator private (version: Http4sVersion)(implicit Cl: Collec
         authImplementation match {
           case AuthImplementation.Disable => None
           case AuthImplementation.Simple =>
-            securityRequirements.map(
+            securityRequirements.map({
               sr =>
-                if (sr.optional)
-                  (LanguageParameter.fromParam(param"authContext: Option[$authContextTypeName]"), sr)
-                else
-                  (LanguageParameter.fromParam(param"authContext: Either[${Term.Name(resourceName)}.$authErrorTypeName, $authContextTypeName]"), sr)
-            )
+                val inner = if (sr.optional) t"Option[$authContextTypeName]" else authContextTypeName
+                (LanguageParameter.fromParam(param"authContext: Either[${Term.Name(resourceName)}.$authErrorTypeName, $inner]"), sr)
```

Comment:
I would change the type to `Either[Forbidden, Option[AuthContext]]` to eliminate the inconsistent state